python-sdk/src/astro/databases/snowflake.py

```diff
@@ -333,6 +435,59 @@ def drop_stage(self, stage: SnowflakeStage) -> None:
     # Table load methods
     # ---------------------------------------------------------
 
+    def is_native_autodetect_schema_available(  # skipcq: PYL-R0201
```

Comment:
@feluelle can we keep this method at the base database?

Reply:
It is, but without implementation as the implementation could slightly differ, I guess.